tds/src/test/java/thredds/servlet/TestPathMatcher.java

```diff
@@ -27,19 +27,20 @@ public class TestPathMatcher {
       "/thredds/dods/tester", "/thredds/dods/short", "/actionable", "myworld", "mynot", "ncmodels", "ncmodels/bzipped"};
 
   private final String path;
-  private final boolean hasMatch;
+  private final String match;
   private static DataRootPathMatcher matcher;
 
-  public TestPathMatcher(String path, boolean hasMatch) {
+  public TestPathMatcher(String path, String match) {
     this.path = path;
-    this.hasMatch = hasMatch;
+    this.match = match;
   }
 
   @Parameterized.Parameters(name = "{0}")
   public static Object[][] getTestParameters() {
-    return new Object[][] {{"nope", false}, {"/thredds/dods/test", true}, {"/thredds/dods/test/lo", true},
-        {"/thredds/dods/test/longer/donger", true}, {"myworldly", true}, {"/my", false}, {"mysnot", false},
-        {"ncmodels/canonical", true}};
+    return new Object[][] {{"nope", null}, {"/thredds/dods/test", "/thredds/dods/test"},
```

Comment:
maybe we can add one more test case to make sure it is matching the longest datasetRoot (don't think that is there already)-- like for roots `datasetRoot` and `datasetRoot2` that path `datasetRoot2/stuff` matches the second root

Reply:
or maybe that isn't needed-- Does the path `datasetRoot2/stuff` match the root `datasetRoot` now?